### DISCUSSION TITLE
How do you remove episodes from the 'Next Up' section without marking it as watched?

### DISCUSSION THREAD

**Discussion by KaXaSA** [jellyfin/jellyfin#8640] [Q&A]
I have some episodes stuck on the **Next Up** that I didn't watch, but I'm planning to -it may take a while- and it's still being released so marking it as _watched_ wouldn't really fix the issue.

So, is there any way to remove the episode/serie from the Next Up section without having to mark it as watched?

(thanks)

**Comment by Dnkhatri** [comment]
Their is an option to only show only recent watched shows. So if you set it to 30 days or something they will disappear from Next Up if not watched in the last 30 days

**Reply by KaXaSA** [reply]
> Their is an option to only show only recent watched shows. So if you set it to 30 days or something they will disappear from Next Up if not watched in the last 30 days

Thanks, found it. For anyone wondering you can find that option in:
`☰ > Settings > Display > Next up > Max days in 'Next Up' `

I still wish we could just right click > remove from next up (until I open that episode again), but this is the next best thing, so it works 👍

**Comment by infectiousstupidity** [comment]
Anyone know if this will be implemented in the future?

**Reply by michaelkrieger** [reply]
I often mark the whole show as watched (click the check on the general show page) and then unwatched. It then removes it from any progress, next up, etc

**Comment by Darkhost1999** [comment]
Solution in comments and what's nice is the setting although applied on PC does apply for tested -> Android Smart TV and Android phone.

**Reply by nielsvanvelzen** [reply]
User settings -> Home -> Uncheck "Display in home screen sections such as 'Latest Media' and 'Continue Watching'"

**Reply by Darkhost1999** [reply]
Thanks from a computer I found the option exactly where you said

![Screenshot_20230928_081323_Moonlight.jpg](https://github.com/jellyfin/jellyfin/assets/60384196/7e06db62-6dbd-455a-b7e9-423d09f1e6a9)

**Reply by Darkhost1999** [reply]
Previously I used to still see my Shows in the Next Up Section. However, now I do not when I disable this setting. There appears to be no more next up category at all anymore. The setting clearly states which categories to remove from and next up is not 1 of those.
I just want separated movies and shows continue watching, always start shows from beginning of episode

**Comment by krdee1** [comment]
I had this issue as well, sometimes I would just open a show to test it out, and then it would linger in my "next up" section forever.

I found that by marking the whole show as watched, returning to the home page, and the searching for it and unmarking it as watched, it would not reappear in the "next up" section while still accurately reflecting my watched/unwatched status. Rarely, I had to try this more than once before it worked.

**Comment by stn777rzv** [comment]
I subscribe to this thread. 
There's two features that should be added:
1. A right click option to instantly remove from Up Next (only to be re-added if a new "Viewing" is done on the same series/media)
2. The ability to exclude libraries or collections from Up Next (eg. i sometimes watch one episode of some cartoon for my kid to assess if its appropriate, i don't want the kids library to be showing up in Up Next in my profile)
